sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -342,6 +342,7 @@ public partial class DataLakeSharedKeyCredentialEntity : Azure.AI.MetricsAdvisor
     public abstract partial class DataSourceCredentialEntity
     {
         internal DataSourceCredentialEntity() { }
+        public Azure.AI.MetricsAdvisor.Models.DataSourceCredentialKind CredentialKind { get { throw null; } }
```

Comment:
are there any samples where you can show this property?

Reply:
We don't have samples for the credential API yet (https://github.com/Azure/azure-sdk-for-net/issues/21586), but we'll have them today, hopefully.